src/tests/components/experiments/ExperimentList.test.js

```diff
@@ -35,9 +39,13 @@ describe("Experiment list component", () => {
   it("should mount", () => {
     const mockStore = configureStore([thunk]);
     const store = mockStore(
-      Map({
-        experiments: Map({
-          items: List([])
+      fromJS({
```

Comment:
i should've been clearer you only need to wrap the whole vanilla JS object in `fromJS`. It does a deep conversion to immutable objects:
```js
      fromJS({
        experiments: {
          items: []
        },
        dates: {
          start_date: Date.parse("2018-08-22"),
          end_date: Date.parse("2019-01-20"),
        }
      })
```

Reply:
Oh whoops sorry! I went back and I think I got all of them cleaned up now!